androidTest/java/com/owncloud/android/utils/AccountsManager.java

```diff
@@ -97,4 +97,18 @@ public static void deleteAllAccounts(Context context) {
             }
         }
     }
+
+
+    //To regularize the URL to build the URL inserted in the account device
+    public static String regularURL(String url){
+        String url_regularized = null;
+        if (url.startsWith("https://")){
+            url_regularized = url.substring(8, url.length()); //skipping the protocol
+        } else if (url.startsWith("http://")) {
```

Comment:
You can create a constant to include the "http://" string and named as **HTTP_SCHEME** for instance.